velox/exec/VectorHasher.h

```diff
@@ -45,6 +45,11 @@ class UniqueValue {
     }
   }
 
+  explicit UniqueValue(Date value) {
+    size_ = sizeof(Date);
```

Comment:
I think this should be sizeof(int64_t). sizeof(Date) works by chance, since it happens to be equal to sizeof(int64_t).

Reply:
I don't fully understand the reasoning behind making it sizeof(int64_t). Please can you give more explanation.

Reply:
size_ is expected to specify the number of valid bytes in data_. You are storing a int64_t (value.days()) in data_, hence, the size of that is sizeof(int64_t). It happens so that sizeof(Date) == sizeof(int64_t), but that's just a coincidence.

Reply:
I see. Thanks for the explanation.